### PR TITLE
Fix NPE when explain anlayze index join

### LINKED ISSUE #8432: Null pointer exception throw when explain analyze query contains index join
If the join type is index join, explain analyze fails with NPE even though query will run fine:

```
java.lang.NullPointerException
        at com.facebook.presto.sql.planner.planPrinter.PlanNodeStatsSummarizer.getPlanNodeStats(PlanNodeStatsSummarizer.java:151)
        at com.facebook.presto.sql.planner.planPrinter.PlanNodeStatsSummarizer.lambda$aggregatePlanNodeStats$0(PlanNodeStatsSummarizer.java:51)
        at java.util.stream.ReferencePipeline$7$1.accept(ReferencePipeline.java:267)
        at java.util.stream.ReferencePipeline$3$1.accept(ReferencePipeline.java:193)
        at java.util.Spliterators$ArraySpliterator.forEachRemaining(Spliterators.java:948)
        at java.util.stream.AbstractPipeline.copyInto(AbstractPipeline.java:481)
        at java.util.stream.AbstractPipeline.wrapAndCopyInto(AbstractPipeline.java:471)
        at java.util.stream.ReduceOps$ReduceOp.evaluateSequential(ReduceOps.java:708)
        at java.util.stream.AbstractPipeline.evaluate(AbstractPipeline.java:234)
        at java.util.stream.ReferencePipeline.collect(ReferencePipeline.java:499)
        at com.facebook.presto.sql.planner.planPrinter.PlanNodeStatsSummarizer.aggregatePlanNodeStats(PlanNodeStatsSummarizer.java:52)
        at com.facebook.presto.sql.planner.planPrinter.PlanPrinter.textDistributedPlan(PlanPrinter.java:200)
        at com.facebook.presto.operator.ExplainAnalyzeOperator.getOutput(ExplainAnalyzeOperator.java:154)
        at com.facebook.presto.operator.Driver.processInternal(Driver.java:303)
        at com.facebook.presto.operator.Driver.lambda$processFor$6(Driver.java:234)
        at com.facebook.presto.operator.Driver.tryWithLock(Driver.java:537)
        at com.facebook.presto.operator.Driver.processFor(Driver.java:229)
        at com.facebook.presto.execution.SqlTaskExecution$DriverSplitRunner.processFor(SqlTaskExecution.java:623)
        at com.facebook.presto.execution.executor.PrioritizedSplitRunner.process(PrioritizedSplitRunner.java:162)
        at com.facebook.presto.execution.executor.TaskExecutor$TaskRunner.run(TaskExecutor.java:463)
        at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
        at java.lang.Thread.run(Thread.java:748)
```

### PR DESCRIPTION
**Pull request by hellium01**
Fixes #8432

### CODE REVIEW

**Comment by electrum** [comment]
The commit message seems to have two typos.

**Comment by hellium01** [comment]
Sorry, David, corrected it.

File: presto-main/src/main/java/com/facebook/presto/sql/planner/planPrinter/PlanNodeStatsSummarizer.java [hunk 1]
```diff
@@ -148,15 +148,15 @@ private PlanNodeStatsSummarizer() {}
             stats.add(new PlanNodeStats(
                     planNodeId,
                     new Duration(planNodeWallMillis.get(planNodeId), MILLISECONDS),
-                    planNodeInputPositions.get(planNodeId),
-                    succinctDataSize(planNodeInputBytes.get(planNodeId), BYTE),
+                    planNodeInputPositions.getOrDefault(planNodeId, 0L),
```
Comment (kokosing) [inline presto-main/src/main/java/com/facebook/presto/sql/planner/planPrinter/PlanNodeStatsSummarizer.java]: I am not sure if it is the way to go. It looks like you are just hiding the real issue here.

@szymonm Can you please take a look why stats are not calculated properly for index join source?

Comment (kokosing) [inline presto-main/src/main/java/com/facebook/presto/sql/planner/planPrinter/PlanNodeStatsSummarizer.java]: Together with @szymonm we did some short investigation. It looks as it is related with the fact that index join source is processed by separate driver, and so stats collected by that driver are collected by main driver. 

Also I am not sure that index join source driver stats are merge-able with stats of the main driver.

Comment (hellium01) [inline presto-main/src/main/java/com/facebook/presto/sql/planner/planPrinter/PlanNodeStatsSummarizer.java]: Thanks kokosing for investigating this. In that case, I will close this PR.

**Pull request closed without merging**